packages/material-ui-lab/src/SliderStyled/SliderStyled.js

```diff
@@ -282,7 +282,7 @@ SliderRoot.propTypes = {
   }),
 };
 
-const Slider = React.forwardRef(function Slider(inputProps, ref) {
+const SliderStyled = React.forwardRef(function SliderStyled(inputProps, ref) {
```

Comment:
Maybe keep the previous name as it will soon replace the v4 Slider component?

Reply:
Agreed, let's rename to Slider.